tests/test_snapshot.py

```diff
@@ -77,8 +77,7 @@ def test_track_writes(self):
         info = elastio_snap.info(self.minor)
         start_nr = info["nr_changed_blocks"]
 
-        with open(testfile, "w") as f:
-            f.write("The quick brown fox")
+        util.dd("/dev/urandom", testfile, 1, bs="1M")
```

Comment:
It will be good to add a comment, that the written bits should be greater than the block size for the test consistency.

Reply:
I don't think it is necessary. There are no 100% reproduce or no-reproduce conditions. Theoretically, this may happen with 1 MB of data as well. But the probability will be lesser.

Reply:
I assume this file is new and probability of the sectors reuse tends to 0 without previous deletion of something, particularly in case of 1 MB.

Reply:
Yes, exactly.